parcel_test.go

```diff
@@ -30,59 +33,94 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db")
+	require.NoError(t, err)
+	defer db.Close()
+
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
-
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
-
-	// delete
-	// удалите добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что посылку больше нельзя получить из БД
+	// Add
+	id, err := store.Add(parcel)
+	require.NoError(t, err)
+	require.NotEqual(t, 0, id)
+
+	// Get
+	storedParcel, err := store.Get(id)
+	require.NoError(t, err)
+	assert.Equal(t, parcel.Client, storedParcel.Client)
+	assert.Equal(t, parcel.Status, storedParcel.Status)
+	assert.Equal(t, parcel.Address, storedParcel.Address)
+	assert.Equal(t, parcel.CreatedAt, storedParcel.CreatedAt)
+
+	// Delete
+	err = store.Delete(id)
+	require.NoError(t, err)
+
+	// Check deletion
+	_, err = store.Get(id)
+	assert.Error(t, err)
```

Comment:
Можно так же проверить тип ошибки
```suggestion
	assert.ErrorIs(t, err, sql.ErrNoRows)
```